.github/workflows/push_trigger.yml

```diff
@@ -64,8 +60,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-          SONAR_ORGANIZATION: ${{ secrets.ORG_KEY }}
         run: |
           cd client
           chmod +x gradlew
-          ./gradlew build test testDebugUnitTestCoverage sonarqube --info --warning-mode all
+          ./gradlew build sonarqube --info --warning-mode all
```

Comment:
Why is the test removed?

Reply:
Do we need both the tests? In that case, will put `test testDebugUnitTestCoverage` back.

Reply:
can you revert this as well

Reply:
![image](https://github.com/mosip/android-registration-client/assets/56625754/3e9a3c4a-c752-4bc7-822b-04da511c1ce8)
This is what I see from my local latest update